ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/snapsync/request/StorageRangeDataRequest.java

```diff
@@ -95,7 +94,6 @@ protected int doPersist(
       final SnapSyncConfiguration snapSyncConfiguration) {
 
     // search incomplete nodes in the range
-    final AtomicInteger nbNodesSaved = new AtomicInteger();
```

Comment:
@matkt FYI, Can this line be removed? As this variable was not modified in the code.

Reply:
Instead of returning 0 I think we need to increment it after each put as we are doing for AccountRange. https://github.com/hyperledger/besu/blob/c622213b976580a247def32e469b1541f472243b/ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/snapsync/request/AccountRangeDataRequest.java#L124
it seems we forgot to add it

Reply:
Got it, I will add it back into this PR.